Makefile

```diff
@@ -261,15 +261,17 @@ lint:
 		echo "--- Linting done. ---"; \
 	fi
 
-	. $(VENVNAME)/bin/activate; \
-	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff || \
-	{ \
-		echo; \
-		echo "------------------------------------------------------------------------------"; \
-		echo "Hint: Apply the required changes."; \
-		echo "      Execute the following command to apply them automatically: make lint_fix"; \
-		exit 1; \
-	} && echo "--- isort check done. ---";
+	if [[  "`git rev-parse --abbrev-ref HEAD`" != "master" ]] && [[ -n "`git diff $(MASTER_BRANCH) --name-only`" ]]; then \
```

Comment:
Do you need the second condition? 
`[[ -n "`git diff $(MASTER_BRANCH) --name-only`" ]]`

Reply:
Aha, this explains it https://github.com/oamg/leapp-repository/runs/4395200390?check_suite_focus=true

Reply:
yep. both are needed.